src/metrics/loc.rs

```diff
@@ -142,6 +139,20 @@ fn init(node: &Node, stats: &mut Stats, is_func_space: bool, is_unit: bool) -> (
     (start, end)
 }
 
+#[inline(always)]
+// This function discriminates among the comments that are next to code lines and
+// the ones that are on a single line. This difference is necessary in
```

Comment:
they are all on a "single" line, we should find a better way to describe them (here and in the name of the variable)